<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ChangePasient</class>
 <widget class="QDialog" name="ChangePasient">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>635</width>
    <height>385</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <widget class="QDialogButtonBox" name="buttonBox">
   <property name="geometry">
    <rect>
     <x>270</x>
     <y>340</y>
     <width>341</width>
     <height>32</height>
    </rect>
   </property>
   <property name="orientation">
    <enum>Qt::Horizontal</enum>
   </property>
   <property name="standardButtons">
    <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
   </property>
  </widget>
  <widget class="QTextEdit" name="FinalDest_T">
   <property name="geometry">
    <rect>
     <x>290</x>
     <y>90</y>
     <width>311</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: #FFFFFF;
border: 2px solid #02142C;
box-sizing: border-box;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_12">
   <property name="geometry">
    <rect>
     <x>-10</x>
     <y>0</y>
     <width>171</width>
     <height>411</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">border-image: url(:/PicResource/Group 28.png);</string>
   </property>
  </widget>
  <widget class="QGraphicsView" name="graphicsView_11">
   <property name="geometry">
    <rect>
     <x>-40</x>
     <y>-60</y>
     <width>711</width>
     <height>491</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true"/>
   </property>
  </widget>
  <widget class="QLabel" name="label_3">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>10</y>
     <width>321</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font-family: Roboto;
font-style: normal;
font-weight: bold;
font-size: 28px;
line-height: 28px;
/* or 70% */

text-align: center;

color: #02142C;
</string>
   </property>
   <property name="text">
    <string>Редактировать</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="label_8">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: Rokkitt;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 22px;
/* identical to box height, or 157% */

display: flex;
align-items: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Номер Рейса</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="NumRace_T">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>90</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: #FFFFFF;
border: 2px solid #02142C;
box-sizing: border-box;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="dateExit">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: #FFFFFF;
border: 2px solid #02142C;
box-sizing: border-box;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QDateTimeEdit" name="TimeExit">
   <property name="geometry">
    <rect>
     <x>340</x>
     <y>160</y>
     <width>111</width>
     <height>21</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: #FFFFFF;
border: 2px solid #02142C;
box-sizing: border-box;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_11">
   <property name="geometry">
    <rect>
     <x>300</x>
     <y>60</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: Rokkitt;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 22px;
/* identical to box height, or 157% */

display: flex;
align-items: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Маршрут</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_12">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: Rokkitt;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 22px;
/* identical to box height, or 157% */

display: flex;
align-items: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Дата</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_13">
   <property name="geometry">
    <rect>
     <x>350</x>
     <y>130</y>
     <width>71</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: Rokkitt;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 22px;
/* identical to box height, or 157% */

display: flex;
align-items: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Время</string>
   </property>
  </widget>
  <widget class="QTextEdit" name="NumPlaz_T">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: #FFFFFF;
border: 2px solid #02142C;
box-sizing: border-box;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_14">
   <property name="geometry">
    <rect>
     <x>190</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: Rokkitt;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 22px;
/* identical to box height, or 157% */

display: flex;
align-items: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Плацкартные места</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QTextEdit" name="NumCompey_T">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>230</y>
     <width>101</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
background: #FFFFFF;
border: 2px solid #02142C;
box-sizing: border-box;
border-radius: 4px;</string>
   </property>
  </widget>
  <widget class="QLabel" name="label_15">
   <property name="geometry">
    <rect>
     <x>327</x>
     <y>190</y>
     <width>91</width>
     <height>31</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">
font-family: Rokkitt;
font-style: normal;
font-weight: normal;
font-size: 11px;
line-height: 22px;
/* identical to box height, or 157% */

display: flex;
align-items: center;

color: #000000;</string>
   </property>
   <property name="text">
    <string>Места Купе</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
   <property name="wordWrap">
    <bool>true</bool>
   </property>
  </widget>
  <zorder>graphicsView_11</zorder>
  <zorder>buttonBox</zorder>
  <zorder>FinalDest_T</zorder>
  <zorder>graphicsView_12</zorder>
  <zorder>label_3</zorder>
  <zorder>label_8</zorder>
  <zorder>NumRace_T</zorder>
  <zorder>dateExit</zorder>
  <zorder>TimeExit</zorder>
  <zorder>label_11</zorder>
  <zorder>label_12</zorder>
  <zorder>label_13</zorder>
  <zorder>NumPlaz_T</zorder>
  <zorder>label_14</zorder>
  <zorder>NumCompey_T</zorder>
  <zorder>label_15</zorder>
 </widget>
 <resources>
  <include location="qrc.qrc"/>
 </resources>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>ChangePasient</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>ChangePasient</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
